node/hyperbahn/rate_limiting.js

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
Same as the comment above. I believe both relay and rate limiting should be part of hyperbahn.